src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -61,7 +61,9 @@ private EppoClient(ConfigurationStore configurationStore, Timer poller, EppoClie
     private Optional<EppoValue> getAssignmentValue(
             String subjectKey,
             String flagKey,
-            SubjectAttributes subjectAttributes) {
+            EppoAttributes subjectAttributes,
+            Map<String, EppoAttributes> assignmentOptions
```

Comment:
The `assignmentOptions` parameter is how we extend the interface for bandits. This is a mapping of string assignment (i.e., action name) to any attributes (i.e., it's context).

Note that this is the private base function. Two public, user-facing functions were added that leverage this. One takes a `Set<String>` for assignment options with no attributes, and the other takes `Map<String, EppoAttributes>` for options with attributes.